### PR TITLE
fixed dropdown mixin issue

### PR DESCRIPTION
**Pull request by LaurelBCodin**
Resolves: https://github.com/zurb/foundation-sites/issues/9836

### CODE REVIEW

File: _vendor/normalize-scss/sass/normalize/_normalize-mixin.scss [hunk 1]
```diff
@@ -424,7 +424,7 @@
     optgroup,
     select,
     textarea {
-      font-family: inherit; /* 1 */
+      font-family: $base-font-family; /* 1 */
```
Comment (IamManchanda) [inline _vendor/normalize-scss/sass/normalize/_normalize-mixin.scss]: Hi @LaurelBCodin This is not needed 
=> https://github.com/zurb/foundation-sites/pull/9973/files
Already added!

File: docs/pages/button.md [hunk 2]
```diff
@@ -40,8 +40,10 @@ Additional classes can be added to your button to change its size and shape.
 <a class="button large" href="#">So Large</a>
 <a class="button expanded" href="#">Such Expand</a>
 <a class="button small expanded" href="#">Wow, Small Expand</a>
+<button class="button test-button">Test button</button>
```
Comment (andycochran) [inline docs/pages/button.md]: Is this intentional, or accidentally left from testing?

File: tmp.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+$button-padding: 4px;
+$button-padding: 4px 2px;
+$button-padding: 4px 1px 3px;
+$button-padding: 4px 5px 3px 1px;
```
Comment (andycochran) [inline tmp.txt]: Maybe this temp file was mistakenly committed?

File: docs/pages/button.md (hunk 2 above)
Comment (LaurelBCodin) [inline docs/pages/button.md]: hey Andy! learning to do a bug fix. yes that and the example thing below were left over from testing but not meant to be pushed.

File: tmp.txt (hunk 3 above)
Comment (LaurelBCodin) [inline tmp.txt]: mistakenly committed - thanks for catching this.

**Comment by LaurelBCodin** [comment]
Thanks for the feedback guys! I updated the PR :) let me know if there is anything else!

**Review by IamManchanda — APPROVED** [review]
I think this is good to go if i am not missing something
cc / @kball for merge!

**Comment by kball** [comment]
Merged!

**Pull request merged**